src/common/app_ble_gap.cpp

```diff
@@ -455,4 +456,50 @@ uint32_t app_ble_gap_adv_set_unregister(uint8_t adv_handle, uint8_t **pp_adv_dat
         return NRF_ERROR_SD_RPC_INVALID_STATE;
     }
 }
+
+int app_ble_gap_adv_buf_register(void * p_buf)
+{
+    if (!app_ble_gap_check_current_adapter_set(REQUEST_REPLY_CODEC_CONTEXT))
```

Comment:
Can we ensure that this function is only called REQUEST_REPLY_CODEC_CONTEXT?

Reply:
i think that is true.